template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+    <td>No, optional</td>
+  </tr>
+</table>
+
+Each template deliverable is explained in the following sections.
+
+## Raw template
+
+Example filename: abc-template.md
+
+The raw template is the basic template for the type of document you are creating.
+It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.
+The template user will copy this template and begin filling it in or editing it as their starting point in the writing process.
+
+### Content and formatting guidelines
+
+Each raw template should have this content:
+- **The template title** - The type of document this template is for. For example, "Tutorial template" or "README template."" This title should be formatted with the heading 1 weight.
+- **Introductory comment** - Include an embedded writing tip at the start that tells users they should first read the accompanying template guide and before-you-start guide before they fill in the template. Put introductory comments in {curly brackets}.
+- **The template sections** - Each section of the temple should begin with a heading with the heading 2 weight. The recommended content and some embedded writing tips fall under the headings.
+- **Embedded writing tips** - You can provide some lightweight writing tips that provide some tips or hints about what kind of content the template user might choose to put in a section of the document. Put embedded writing tips in {curly brackets}.
+
+> :triangular_flag_on_post: **NOTE: The basic content of the raw template depends on the type of document you are creating a template for. Different types of documents can have widely varying content needs. After you have completed your research into this document type and are preparing to create the template itself, you should create headings for each of the main sections of this type of document.**
+
+
+### Frequently asked questions
+
+**Q: Is it better to include all the possible sections that someone could possibly include in the raw template or should it just have the most common sections?**
+
+A: While this is mostly a judgment call on your part as the template author, it might be better to err on the side of being comprehensive.
+Don’t forget that in your template guide you can indicate whether it is common or recommended to include this section or not.
+
+**Q: Can I use heading 3 weights for sub-sections?**
+
+A: Yes, but try not to go much deeper than level 3.
+
+
+## Template guide
+
+Example filename: abc-template-guide.md
+
+This guide provides a deeper explanation of how to fill in the raw template.
+It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document.
+
+In essence, this guide supports the key decisions that the template user might need to make during the writing process.

```

Comment:
Change to: "This guide explains the key decisions that the template user needs to make during the writing process."

Reply:
Sure.

Reply:
I've adopted this suggestion.